ui/src/components/easter/EasterBanner.js

```diff
@@ -0,0 +1,67 @@
+import {
+  Flex,
+  Center,
+  Button,
+  Link,
+  Box
+} from '@chakra-ui/react';
+import { useHistory } from "react-router-dom";
+
+
+const EasterBanner = () => {
+  const isEasterSite = useHistory().location.pathname.includes("easter");
+
+  return (
+    <Flex
+    w="100vw"
+    bgImage={`url(${
+        process.env.PUBLIC_URL + "/images/easter/EasterBanner.png"
+    })`}
+    h="6vh"
+    p={2}
+    bgPosition="center"
+    bgSize="cover"
+    justify="center"
+    display={(isEasterSite) ? "none" : "flex"}
+  >
+    <Flex w="100vw" justify="space-around">
+        <Center>
+          <Link href="/easter" style={{ lineHeight: '0', textDecoration: 'none' }}>
+            <Button
+              h="3.5vh"
+              lineHeight="0"
+              w="95%"
+              p={5}
+              border="0px"
+              backgroundColor="rgba(0,0,0,0)"
+              color="black"
```

Comment:
<img width="672" alt="Screenshot 2022-03-13 at 18 39 58" src="https://user-images.githubusercontent.com/16095743/158055721-e3315904-2a5e-431e-a180-ea6bd637cb4c.png">
Is this hover effect on the button intentional? it looks kinda weird lol